server/src/assets/script.js

```diff
@@ -650,9 +650,17 @@ $(document).ready(
                 }
             });
         });
-        
+
+        // Alternative filechooser button for Driver and Controller
+        const actualBtn = document.getElementById('selectEngineFile');
+        const fileChosen = document.getElementById('file-chosen');
+        actualBtn.addEventListener('change', function(){
+        fileChosen.textContent = this.files[0].name
```

Comment:
- Rewrite this functionality to be inside its own `$('#...').change(function() { ... })` function using jQuery, just like the other functions above it. 
- Cannot find the HTML element with id `selectEngineFile`.
- Use more accurate variable names and HTML ids, e.g., `file-chosen` --> `engine-file-chosen`. 
- Use camel casing for HTML ids.

Reply:
selectEngineFile is in line 57 in driver.html
Since I am using this function for driver.html and controller.html I thought a more broad term would make more sense, since it is used for uploading custom Engines and the interlocking algorithm

Reply:
I would even change selectEngineFile to selectFile